web/picture/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
Mit `.gitignore` werden Dateien von git ignoriert. Hier ignorieren wir alle (`*`) hochgeladenen Bilder in `/web/picture/` außer `user_default.png` (siehe unten, habe ich selbst gemalt - mit TouchPad, darum ists auch so grauenhaft ^^).

Das `!` steht hier wie bei bool für `NOT`. Die `.gitignore` Datei selbst wird logischerweise auch nicht ignoriert :D